test/Tool/more-knownbits11.opt

```diff
@@ -0,0 +1,12 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: FileCheck %s < %t
+
+; CHECK: LGTM
+%0:i8 = var (p)
+%1:i8 = var (z)
+%2:i8 = udivexact %0, %1
+%3:i8 = ctpop %2
+%4:i1 = eq %3, 1:i8
+cand %4 1:i1
```

Comment:
this one might not be testing what you think it is testing, it is still LGTM when the (z) is removed